holidays/countries/south_korea.py

```diff
@@ -190,7 +194,9 @@ def _populate(self, year):
             self._add_with_alt_holiday("Hangeul Day", date(year, OCT, 9))
 
         # Christmas Day
-        self._add_christmas_day("Christmas Day")
+        self._add_with_alt_holiday(
+            "Christmas Day", date(year, DEC, 25), since=2023
+        )
```

Comment:
@DiceMono, thank you for good and on-time update. I just want to suggest this change (so there will be no need for `DEC` constant import):

```suggestion
        name = "Christmas Day"
        dec_25 = self._add_christmas_day(name)
        self._add_with_alt_holiday(
            name, dec_25, add_hol=False, since=2023
        )
```

Reply:
it'd be better. i'll change it.